app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :user
```

Comment:
- [ ] Like the ERD state that the relation between post and comment, and the likes belong to the AuthorID, so in this case the belong to should be author, with class_name user.